pkg/controller/atlasconnection/atlasconnection_controller.go

```diff
@@ -398,6 +398,7 @@ func getOwnedConfigMap(connection *dbaas.MongoDBAtlasConnection, connectionStrin
 			dbaas.ConnectionStringsStandardSrvKey: connectionStringStandardSrv,
 			dbaas.ConnectionStringsStandardKey:    connectionStringStandard,
```

Comment:
are these 2 properties doing anything for us since we're not using them for service binding?

Reply:
@jeremyary we do not use these two right now (as Sandip suggested earlier that we keep them just in case application developers want to use them directly). Should we remove them?

Reply:
@jianrongzhang89 I think, we can remove them now as the host value is already passed in the `host` property.

I just remembered about the `database` and `options` properties as well that was mentioned in the document. Do you want to add those in this PR as well?
![Screen Shot 2021-08-16 at 1 17 08 PM](https://user-images.githubusercontent.com/2349910/129603538-e4b8eee5-8719-4cde-94a0-e0dcf19562eb.png)

Reply:
@jianrongzhang89 to clarify, I was under the assumption that this whole thing is from Atlas operator to Quarkus, but since this is more for MongoDB properties, I don't think anyone would be using these properties (`connectionStringStandardSrv` and `connectionStringStandard`). Please correct me my new assumption is wrong :-)

Reply:
@sgahlot OK i will removeconnectionStringStandardSrv and connectionStringStandard then. For database and  options did you mean that Atlas Operator should populate them with default values?

Right now the operator simply uses "admin" database to generate the DB users. Late one a design is needed to extend the capability.

Reply:
What I meant was that these two new properties are also part of MongoDB binding, and the binding converter would support it, but I don't see those in our operator codebase. Is there a possibility of passing value for these two properties from our operator? If so, then they should be part of our code, otherwise not. I'm not sure how one would get these values (from Mongo server) - at the moment.

From the binding converter perspective, if they are missing, that's ok as they are both optional.

Reply:
For Jianrong/Atlas:
- Please remove `connectionStringStandardSrv` & `connectionStringStandard`
- Please add `srv`
- Please do NOT add  or supply values for `database` or `options` at this time

For Sandip:
- Continue with what we discussed in team call - you SHOULD add `srv`, `database` & `options` even though the last 2 will not be used by Atlas.

The admin db (which Jianrong is referencing) is NOT the same as the default connection (which connection properties could include). Regardless, if you do these 3 steps above, we should be in a good place to continue.

Reply:
Thanks @jeremyary. Sounds good to me.